docs/development/creating-a-first-plugin.md

```diff
@@ -95,10 +107,14 @@ The following methods are inherited from ```Plugin``` and can be used to install
 * ```onStartup()``` - Invoked when the plugin has been installed.
 * ```onUninstall()``` - Invoked when the plugin has been uninstalled.
 * ```isInstalled()``` - Can be invoked to determine if the plugin has been installed yet.
-* ```registerCommand(Command c)``` - Invoke to register your own commands.
-* ```registerListener(EventListener e)``` - Invoke to register your own event listeners.
-* ```unregisterListener(EventListener e)``` - Invoke to remove an event listener.
+* ```registerCommand(Command command)``` - Invoke to register your own commands.
+* ```registerListener(EventListener listener)``` - Invoke to register your own event listeners.
+* ```unregisterListener(EventListener listener)``` - Invoke to remove an event listener.
```

Comment:
Please leave out the argument names. The signature doesn't necessarily has to provide a name for an argument so we'll take that away in order for less mental overhead.